inc/class-wpseo-utils.php

```diff
@@ -195,34 +219,35 @@ public static function trim_recursive( $value ) {
 	 * @return string
 	 */
 	public static function translate_score( $val, $css_value = true ) {
-		if ( $val > 10 ) {
```

Comment:
What I find strange about the original implementation is that it would allow for a 100 point scale value of 9 and evaluate that as a "good" score.